File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_Equipment.java
package com.example.clinicsystem.models.classes;

import com.example.clinicsystem.helpers.DatabaseConnection;
import com.example.clinicsystem.models.enums.EquipmentType;

public class M_Equipment {
    private String name;
    private EquipmentType equipmentType;
    private int price, id, quantity;
    private boolean isAvailable;

    public M_Equipment(String name, EquipmentType equipmentType, int price, int id, int quantity, boolean isAvailable) {
        this.name = name;
        this.equipmentType = equipmentType;
        this.price = price;
        this.id = id;
        this.quantity = quantity;
        this.isAvailable = isAvailable;
    }

    public M_Equipment() {
    }

    public int addEquipment() {
        DatabaseConnection databaseConnection = DatabaseConnection.getINSTANCE();

        int available;

        if(this.isAvailable) {
            available = 1;
        } else {
            available = 0;
        }

        return databaseConnection.insertInto("equipment", "(Equipment_IsAvailable, Equipment_Name, Equipment_Price, " +
                "Equipment_Quantity, Equipment_Type)", "('" + available + "', '" + this.name + "', '" + this.price +
                "', '" + this.quantity + "', '" + this.equipmentType.getName() + "')");
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        name = name;
    }

    public EquipmentType getEquipmentType() {
        return equipmentType;
    }

    public void setEquipmentType(EquipmentType equipmentType) {
        this.equipmentType = equipmentType;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public boolean isAvailable() {
        return isAvailable;
    }

    public void setAvailable(boolean available) {
        isAvailable = available;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: clinic/M_Equipment.php
<?php
include "DatabaseConnection.php";

class M_Equipment {
	private $name;
    private $equipmentType;
    private $price, $id, $quantity;
    private $isAvailable;

    public function __construct($name, $equipmentType, $price, $id, $quantity, $isAvailable) {
        $this->name =          $name;
        $this->equipmentType = $equipmentType;
        $this->price =         $price;
        $this->id =            $id;
        $this->quantity =      $quantity;
        $this->isAvailable =   $isAvailable;
    }

    public function addEquipment() {
        $conn = DatabaseConnection::getINSTANCE();
		
		$available = 0;
		
		if($this->isAvailable == TRUE) {
			$available = 1;
		}
		
		return $conn->insertInto("equipment", "(Equipment_Type, Equipment_Name, Equipment_Price, 
			Equipment_Quantity, Equipment_IsAvailable)", "('". $this->equipmentType . "', '" .
			$this->name . "', '" . $this->price . "', '" . $this->quantity . "', '" . $available . "')");
    }
	
	public function getAllEquipment($condition) {
        $conn = DatabaseConnection::getINSTANCE();
		
		$result = $conn->select("equipment", $condition);
		$equipments = array();
		
		while ($row = $result->fetch_assoc()) {
			$available = FALSE;
			
			if($row["Equipment_IsAvailable"] == 1) {
				$available = TRUE;
			}
			
			$equipment = new M_Equipment($row["Equipment_Name"], $row["Equipment_Type"],
						$row["Equipment_Price"], $row["Equipment_ID"],
						$row["Equipment_Quantity"], $available);
			array_push($equipments, $equipment);
		}
		
		return $equipments;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getEquipmentType() {
        return $this->equipmentType;
    }

    public function setEquipmentType($equipmentType) {
        $this->equipmentType = $equipmentType;
    }

    public function getPrice() {
        return $this->price;
    }

    public function setPrice($price) {
        $this->price = $price;
    }

    public function isAvailable() {
        return $this->isAvailable;
    }

    public function setAvailable($available) {
        $this->isAvailable = $available;
    }

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getQuantity() {
        return $this->quantity;
    }

    public function setQuantity($quantity) {
        $this->quantity = $quantity;
    }
}

?>